.editorconfig

```diff
@@ -149,7 +149,8 @@ dotnet_public_api_analyzer.require_api_files = true
 
 # IDE0055: Fix formatting
 # Workaround for https://github.com/dotnet/roslyn/issues/70570
-dotnet_diagnostic.IDE0055.severity = warning
+# Disabled because the current SDK warns on collection expressions
+dotnet_diagnostic.IDE0055.severity = none
```

Comment:
Is it already fixed in the sdk? If not, is there a bug? If so, is updating the SDK an option?

Reply:
it's fixed in the SDK.  however, we cannot update the SDK for reasons that are never quite clear to me.  so we're stuck in this state.  Being able to dogfood trumps using an old SDK though, so this has to happen.